src/actions/item.js

```diff
@@ -32,6 +32,14 @@ module.exports = {
     }
   },
 
+  copy(payload, meta) {
```

Comment:
To re-use actions we should use the respective functions instead of creating actions with the same type in multiple places (that helps later if we change `meta` attributes for example). So `copy` could call `export(payload, { target: ':clipboard:', ...meta })`.

However, in this case I would prefer to just use `export` directly and calling `onItemExport(items, { target: ':clipboard:' })` in the component -- this has the advantage that we have to pass down only the export function to the component which will pay off later if we use different shortcuts to call the export with different presets / plugins. We may also add an 'item.copy' later which would be unrelated to export.

Reply:
I've changed that, but to be honest I don't understand the abstractions involved very much, so maybe it's not perfect. We'll fix it when there's a need for a separate `copy` command.